### ISSUE TITLE
Custom CSS not completely loading

### ISSUE THREAD

**Issue by Eruasa** [thecodemonkey/synthwave-x-fluoromachine-epic-animations#14]
The colors are getting loaded, but the rest like the background or the glowing is not getting loaded.

Checked if I had syntax errors from copying wrong, but that's not the case.
I also reinstalled the theme, but that didn't change a thing.

**Comment by thecodemonkey** [comment]
In the current version it is not enough to click on the Install button. You have to perform additional manual steps. See readme/installation. Did you follow the installation instructions?

**Comment by Eruasa** [comment]
If you mean the linking in the settings.json, then yes, I did that.
Weird thing is that I also have the theme installed on my pc at my work and there it works, so no idea why it doesn't work back at my home pc.

**Comment by thecodemonkey** [comment]
Yes sounds weird. Normally it is different, on the work computer nothing works but on the private one. Different OS Windows/Mac? Different VS code versions? Run VS Code as Admin?

The installation is completely MacGyver style. I inherited it that way and always wanted to improve it.

**Comment by Eruasa** [comment]
No, both Win 10, but don't know right now if it's the same OS version and VS Code version.
Running it as Admin makes no difference.

**Comment by sangoi-exe** [comment]
I know it's been months since this issue was opened, but yesterday I had the same problem and ended up finding the solution. It turns out there is no global settings.json file for VS Code; each workspace has its own. If you go to the command palette and type settings.json, the "workspace settings" option will be for your current workspace, so just put the paths to the Custom CSS files there.

**Comment by JGLAHE** [comment]
I also just want to mention the obvious when pasting the example paths into your settings.json (which I initially failed to do): make sure those paths are correct for your setup!

- make sure the `file:///`... is still prepended to the absolute path
- replace `{your username}` with your actual username folder name
- (On Windows) make sure the drive letter is correct (my VSCode isn't installed at C:)
